### PR TITLE
Pcdcp one second factory ingest

### PR DESCRIPTION
**Pull request by aclaycomb-usgs**

### CODE REVIEW

File: geomagio/pcdcp/PCDCPFactory.py [hunk 1]
```diff
@@ -54,20 +54,15 @@ def parse_string(self, data, **kwargs):
         parser = PCDCPParser()
         parser.parse(data)
 
-        yr = int(parser.header['year'])
-        yrday = int(parser.header['yearday'])
+        sample_periods = {4: 60.0, 5: 1.0}
```
Comment (erigler-usgs) [inline geomagio/pcdcp/PCDCPFactory.py]: This is a little obscure. Insert a brief comment explaining why the dictionary keys are '4' and '5'.

**Comment by erigler-usgs** [comment]
This looks good once you add the requested comment on the logic used to differentiate minute from second PCDCP files. I just want to point out something related to the perceived need to explicitly name the channels in the test routines for consistency between OS's.

The PCDCPParser._post_process() method creates a dictionary with keys named after the channels created in PCDCPParser._set_channels(). Dictionary keys are **not** guaranteed to be stored in the order they were created in Python. HOWEVER, ObsPy Streams, which are what are used in the test routines, are not based on dictionaries, but are rather glorified **lists* of ObsPy Traces.

So, if one wished to avoid such problems in the future, they would not store data read in by the Parser as a dictionary indexed by keys, but rather as a list of lists. After all, the channel names are already stored in the Parser object, and in the desired order.

I'm not proposing that you do this, I just wanted to be sure we understood the source of the discrepancies that arose when using Linux to test code developed on MacOS.

File: geomagio/pcdcp/PCDCPFactory.py [hunk 2]
```diff
@@ -53,20 +54,17 @@ def parse_string(self, data, **kwargs):
         parser = PCDCPParser()
         parser.parse(data)
 
-        yr = int(parser.header['year'])
-        yrday = int(parser.header['yearday'])
+        # minutes files times are 4 characters long (1440)
+        # seconds files times are 5 characters long (86400)
```
Comment (aclaycomb-usgs) [inline geomagio/pcdcp/PCDCPFactory.py]: added these lines to address dictionary for selecting seconds/minutes

File: geomagio/pcdcp/PCDCPFactory.py (hunk 1 above)
Comment (aclaycomb-usgs) [reply geomagio/pcdcp/PCDCPFactory.py]: https://github.com/usgs/geomag-algorithms/pull/246/files#r288666760

File: geomagio/pcdcp/PCDCPFactory.py (hunk 2 above)
Comment (erigler-usgs) [reply geomagio/pcdcp/PCDCPFactory.py]: Perfect.

**Pull request merged**